ansible_rulebook/util.py

```diff
@@ -39,9 +40,12 @@ def get_horizontal_rule(character):
 
 
 def render_string(value: str, context: Dict) -> str:
-    return jinja2.Template(value, undefined=jinja2.StrictUndefined).render(
-        context
-    )
+    if "{{" in value and "}}" in value:
```

Comment:
This is to ensure that we don't get burnt by the ast.literal_eval which will convert the integer wrapped as a string to integer.
https://jinja.palletsprojects.com/en/3.0.x/nativetypes/#jinja2.nativetypes.NativeTemplate.render